src/include/distributed/causal_clock.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * causal_clock.h
+ *
+ * Data structure definitions for managing hybrid logical clock and
+ * related function declarations.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef CAUSAL_CLOCK_H
+#define CAUSAL_CLOCK_H
+
+#include "distributed/type_utils.h"
+
+/*
+ * Clock components - Unsigned 64 bit <LC, C>
+ * Logical clock   (LC): 42 bits
+ * Counter          (C): 22 bits
```

Comment:
Same as the readme.

The observation of unused macro's seem to corroborate that we have an increased bitspace for the logical clock

Reply:
Both the macros are used in the code.
![image](https://user-images.githubusercontent.com/44680808/198688362-aa50d054-2009-4522-94db-c243e47534e6.png)